### DISCUSSION TITLE
Problem with some tif images in Cantaloupe server

### DISCUSSION THREAD

**Discussion by rita-funk** [cantaloupe-project/cantaloupe#632] [Q&A]
Hello, for some large tif images that I want to use in the Cantaloupe server, I got the following error, when I use it from the client:
Failed to load resource: the server responded with a status of 500 (Server Error)

**Comment by DiegoPino** [comment]
Hi Rita,

Chances are your Max Pixel settings are limiting the response for those images or you are running out of Memory. Please enable either error output on 500 error messages on your site or check your logs. All these are settings in your properties file
